tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "module": "commonjs",
     "target": "es6",
     "declaration": true,
-    "strict": true
+    "strict": true,
+    "esModuleInterop": true
```

Comment:
Could you explain why this is needed?

Reply:
This was needed for the `@ungap/url-search-params` polyfill, but no longer needed since we're removing it!